docs/2-the-zarf-cli/100-cli-commands/zarf_tools_helm_dependency.md

```diff
@@ -38,7 +38,7 @@ appending '/index.yaml' to the URL, it should be able to retrieve the chart
 repository's index. Note: 'repository' can be an alias. The alias must start
 with 'alias:' or '@'.
 
-Starting from 2.2.0, repository can be defined as the path to the directory of
+A repository can be defined as the path to the directory of
```

Comment:
This is helm 2.2.0 right? We're bringing in helm anyway and 2.2.0 was released 7 years ago so it seems unnecessary to add this. This one is just my opinion though. Does the users version of helm ever matter when their using zarf?

Reply:
We can edit this message for Zarf though this whole file originally came from the Helm project hence why it is geared towards Helm a bit more - TLDR, Helm currently has their cmd stuff in their main package and it isn't importable (even with //go:linkname)

Reply:
Ah in that case maybe we say `starting from helm 2.2.0`, I just felt it read a bit awkwardly

Reply:
Oh it's actually copied by `make docs-and-schema ` anyway. Probably not a big enough deal to change